Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles/Definitions/FileTransporter.cs

```diff
@@ -13,321 +13,319 @@
 using Frends.FTP.DownloadFiles.TaskConfiguration;
```

Comment:
https://github.com/FrendsPlatform/Frends.FTP/blob/3b2b2fbd96e793fb58e0ddc3e006dc3f6fcb985c/Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles/Definitions/FileTransporter.cs#L2-L5
You are not using these namespaces, so they can be removed.

https://github.com/FrendsPlatform/Frends.FTP/blob/3b2b2fbd96e793fb58e0ddc3e006dc3f6fcb985c/Frends.FTP.DownloadFiles/Frends.FTP.DownloadFiles/Definitions/FileTransporter.cs#L10
You are not using this namespace, so it can be removed.